src/lib/pubkey/frodokem/frodokem.h

```diff
@@ -0,0 +1,105 @@
+/*
+* FrodoKEM implementation
+* Based on the MIT licensed reference implementation by the designers
+* (https://github.com/microsoft/PQCrypto-LWEKE/tree/master/src)
+*
+* The Fellowship of the FrodoKEM:
+* (C) 2023 Jack Lloyd
+*     2023 René Meusel, Amos Treiber - Rohde & Schwarz Cybersecurity
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_FRODOKEM_H_
+#define BOTAN_FRODOKEM_H_
+
+#include <botan/frodo_mode.h>
+#include <botan/pk_keys.h>
+
+#include <tuple>
+#include <vector>
+
+namespace Botan {
+
+class FrodoKEM_PublicKeyInternal;
+class FrodoKEM_PrivateKeyInternal;
+
+/**
+ * FrodoKEM is a unstructured lattice-based post-quantum secure KEM. It is a
```

Comment:
```suggestion
 * FrodoKEM is an unstructured lattice-based post-quantum secure KEM. It is a
```
(sorry for this nitpicking 😝)